content_restrictions/conf/locale/en/LC_MESSAGES/text.po

```diff
@@ -77,62 +77,104 @@ msgid ""
 msgstr ""
 
-#: content_restrictions.py:82
+#: content_restrictions.py:78
 msgid "User Provided Password"
 msgstr ""
 
-#: content_restrictions.py:84
+#: content_restrictions.py:80
 msgid ""
 "The password that the user has provided. This is used to check if the user "
 "has entered the correct password."
 msgstr ""
 
-#: content_restrictions.py:92
+#: content_restrictions.py:88
 msgid "IP Restriction"
 msgstr ""
 
-#: content_restrictions.py:94
-msgid ""
-"When enabled, the content will be restricted by IP address."
+#: content_restrictions.py:89
+msgid "When enabled, the content will be restricted by IP address."
 msgstr ""
 
-#: content_restrictions.py:101
+#: content_restrictions.py:95
 msgid "IP Whitelist"
 msgstr ""
 
-#: content_restrictions.py:103
+#: content_restrictions.py:97
 msgid ""
 "List of IP addresses or IP address ranges from which the content can be "
 "accessed. The compatible protocols are IPv4 and IPv6. If the learner's IP "
 "address is not in the whitelist, they will not have access to the content. "
 "The range should be in the CIDR format, e.g. (IPv4) \"192.168.1.0/24\", in "
 "this case only IP addresses from  \"192.168.1.0\" to \"192.168.1.255\" can "
 "access to the content, or (IPv6) \"2001:db8:85a3::/48\", in this case only "
-"IP addresses from \"2001:db8:85a3::\" to "
-"\"2001:db8:85a3:ffff:ffff:ffff:ffff:ffff\" can access to the content. Also, "
-"a single IP address can be provided, e.g. (IPv4) \"172.16.0.0\", or (IPv6) "
-"\"7ac:264a:dd69::\". Alternatively, is possible mix IP addresses and IP "
-"ranges with different formats, e.g. [\"192.168.1.0/24\", \"172.16.0.0\", "
-"\"65c1:e700::/24\", \"5c87::\"]"
+"IP addresses from \"2001:db8:85a3::\" to \"2001:db8:85a3:ffff:ffff:ffff:ffff:"
+"ffff\" can access to the content. Also, a single IP address can be provided, "
+"e.g. (IPv4) \"172.16.0.0\", or (IPv6) \"7ac:264a:dd69::\". Alternatively, is "
+"possible mix IP addresses and IP ranges with different formats, e.g. "
+"[\"192.168.1.0/24\", \"172.16.0.0\", \"65c1:e700::/24\", \"5c87::\"]"
 msgstr ""
 
-#: content_restrictions.py:119
+#: content_restrictions.py:113
 msgid "IP Restriction Explanatory Message"
 msgstr ""
 
-#: content_restrictions.py:121
+#: content_restrictions.py:114
 msgid ""
 "This message will be shown when the learner is browsing from an IP address "
 "that is not whitelisted."
 msgstr ""
 
-#: content_restrictions.py:126
+#: content_restrictions.py:117
 msgid ""
 "You are entering from an IP address that is not whitelisted. Please follow "
 "the instructions provided in the course introduction video."
 msgstr ""
 
-#: content_restrictions.py:397
+#: content_restrictions.py:123
+msgid "SEB Restriction"
+msgstr ""
+
+#: content_restrictions.py:125
+msgid ""
+"When enabled, the content will be restricted to be accessed only from Safe "
+"Exam Browser. To use this restriction, it is recommended to have the setting "
+"'SEB_INDIVIDUAL_COURSE_ACTIVATION = True' in the platform. This setting "
```

Comment:
Questions:
1. Is mentioning this setting relevant for instructors? Should we only mention that SEB individual course activation should be enabled? So we don't include operational docs here.
2. Same question for the seb-openedx plugin.

Reply:
Configuration details are not necessary as the instructors are not in the capacity to do so. I updated the help message, what do you think now?: https://github.com/eduNEXT/xblock-content-restrictions/commit/6636eb27dada21744f2081d3fb677dda19c4174f

Reply:
I like it. Thanks!